src/components/Player.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const Player = (props) => (
+<table>
+    <thead>

```

Comment:
Minor, but isn't this a table row? why are you wrapping this with table headers (thead) and `<table />`? I think returning just the `<tr>` should be enough. Likewise, if these are not headers, maybe a `<td>` is more accurate than `<th>` (Here you check how html tables usually looks like https://html5-tutorial.net/tables/thead-tbody/)

```suggestion
        <tr>
            <td>{props.name}</td>
            <td>{props.position}</td>
            <td>{props.jerseyNumber}</td>
            <td>{props.nationality}</td>
        </tr> 
```

Reply:
@wurm90 Appreciate it, many thanks!  changes in the table already solved and pushed into branch 👍